nengo/simulator.py

```diff
@@ -55,6 +55,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    unsupported = []
+    """Defines features unsupported by a simulator. The format is a list of
```

Comment:
I recall that this is a convention for documenting variables in `pydoc`, but we don't use it anywhere else, so it feels a little out of place... Should we roll this into the `Simulator` docstring instead? It's missing a lot, so don't worry about making it complete, but just moving this to the class docstring I think would be better as I don't think `Sphinx` reads these.

Also, to render as code, you need two backticks before and after the string in ReST.